README.md

```diff
@@ -100,14 +109,24 @@ curl -sSL --ssl https://github.com/deislabs/mystikos/releases/download/v${LATEST
 export PATH="$PATH:$(pwd)/mystikos/bin"
 ```
 
-## Install From Source
+**Debian package installation**
+```
+# change this to match the latest version
+LATEST='0.7.0'
+RELEASE="mystikos-${LATEST}-x86_64"
```

Comment:
Filename doesn't match the release assets -- missing the OS name e.g. `Ubuntu-1804_mystikos-0.7.0_rc1-x86_64.deb`
(Also _rc1 vs -rc1 difference between the version name and file name would be a problem)

Reply:
I am fine with removing rc1, since we want this for the final release, but the name does not match the package name uploaded.